### ISSUE TITLE
Install CKEditor through Composer

### ISSUE THREAD

**Issue by raeldc** [joomlatools/joomlatools-framework-ckeditor#3]
No need to save the whole editor in the repo. We can install it through Composer.

http://docs.ckeditor.com/#!/guide/dev_package_managers

**Comment by johanjanssens** [comment]
@raeldc True, the problem with this approach is that it will install it in the vendor folder. Vendor folder lives below HTTP root. As a result you will need additional rewrite rules to be able to get to the resources. This is not ideal. 

Other projects solved this by either symlinking or copying the resources to the proper location. We do not have an asset management solution for this yet that can handle this.

**Comment by johanjanssens** [comment]
@ercanozkaya When working on our composer installers can we also add support for custom folders so that we do not need to install CKEditor in the repo? 

See also: https://getcomposer.org/doc/faqs/how-do-i-install-a-package-to-a-custom-path-for-my-framework.md
